mon-pix/translations/fr-fr.json

```
@@ -19,6 +19,12 @@
     "suffix": "Pix"
   },
 
+  "api-error-messages": {
+    "unauthorized-error": "L'adresse e-mail et/ou le mot de passe saisis sont incorrects.",
+    "bad-request-error": "Les données envoyées ne sont pas au bon format.",
```

En tant qu'utilisateur, je ne sais pas si c'est Pix (API) qui a envoyé des données incorrectes ou moi (mon navigateur). En même temps, l'IHM ne devrait pas me permettre d'envoyer des données erronées, et pas sûr que je sache quel le le "bon format", auquel cas le problème doit être géré par les devs. 

Donc deux suggestions:
1) évasive
```suggestion
    "bad-request-error": "Les données que vous avez soumises ne sont pas au bon format.",
```
2) Directe
```suggestion
    "bad-request-error": "Les données envoyées par votre navigateur sont incorrectes. Essayez de les corriger et contactez le support.",
```

---

Vu ensemble, la seule validation coté api porte sur la présence des deux champs. Avec l'introduction du champ required, ce message ne pourra plus être affiché coté front. J'ai profité pour modifier l'api pour qu'elle renvoie le champ manquant dans le détail de l'erreur. 
Coté front, j'ai pris ta première suggestion de message. Y aura surement une amélioration des messages des bad request pour afficher plus de détail sur les violations coté front mais ca sera dans un autre ticket.

---

Du point de vue de l'utilisateur d'une application front (vs. un dev qui interroge l'API), il n'y a pas de différence entre 400 (Bad Request) et 500 (Server Error) : dans les deux cas c'est un problème que le fournisseur de l'application (Pix) doit régler. Tout ce qu'on peut faire quand ça arrive (parce que ça arrivera même si on a essayé de l'éviter…) c'est s'excuser, dire que le problème va être analysé par nos équipes et inviter l'utilisateur à réessayer un peu plus tard puis contacter le support si ça ne marche toujours pas (bizarre de lui dire "Essayez _et_ contactez le support", s'il essaie et que ça marche pourquoi contacterait-il le support ?)